examples/fastapi/advanced.py

```
@@ -0,0 +1,432 @@
+            try:
+                async for chunk in chat_service.process_stream(
+                    message=request.message,
+                    model=request.model,
+                    system_prompt=request.system_prompt,
+                    temperature=request.temperature,
+                    max_tokens=request.max_tokens
+                ):
+                    if chunk.text:
+                        yield f"data: {chunk.text}\n\n"
+                
+                yield f"data: [DONE]\n\n"
```

Flake8 (7.2.0)

[error] f-string is missing placeholders

(F541)